BE/Trusts/Trusts.rdf

```diff
@@ -142,9 +151,9 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>trust agreement</rdfs:label>
-		<skos:definition>Formal agreement through which a trustor vests the ownership rights (title) to one or more assets to one or more trustees for conservation and protection on behalf of one or more beneficiaries of the trust. It normally states the (1) purpose for which the trust was established and fulfillment of which will terminate the trust, (2) details of the assets placed in the trust, (3) powers and limitations of the trustees, their reporting requirements, and other associated provisions, and (4) may also specify the trustees&apos; compensation, if any. A trust agreement involving real estate requires its exact description and the trustor&apos;s express, written consent to create the trust to be valid. A will admitted to probate may also act like a trust agreement.</skos:definition>
-		<skos:editorialNote>See also Deed. These are distinct from Contracts in that they impose obligations but without necessarily reciprocating rights.</skos:editorialNote>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/trust-agreement.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>formal agreement that establishes a trust, whereby the trustor(s) gives the trustee(s) the responsibility to hold and manage assets for the beneficiary(ies)</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom>Barron&apos;s Dictionary of Finance and Investment Terms, Ninth Edition, 2014.</fibo-fnd-utl-av:adaptedFrom>
```

Comment:
See earlier comment about unnecessarily adding references to copyrighted work. The Barron's book (7th Ed) does not even have an entry for Trust Agreement.